pkg/controllers/dynakube/activegate/internal/statefulset/statefulset.go

```diff
@@ -303,6 +304,8 @@ func (statefulSetBuilder Builder) addPersistentVolumeClaim(sts *appsv1.StatefulS
 		}
 		sts.Spec.PersistentVolumeClaimRetentionPolicy = defaultPVCRetentionPolicy()
 	}
+
+	statefulset.SetPVCAnnotation()(sts)
```

Comment:
update unit test? 🥺

Reply:
Updates in [b1fea19](https://github.com/Dynatrace/dynatrace-operator/pull/4645/commits/b1fea19c35d07219fe41522a3d21d47041bd8907)